src/friends.rs

```diff
@@ -122,6 +122,27 @@ impl<Manager> Friends<Manager> {
         }
     }
 
+    pub fn activate_game_overlay_to_store(&self, app_id: AppId) {
+        unsafe {
+            sys::SteamAPI_ISteamFriends_ActivateGameOverlayToStore(
+                self.friends,
+                app_id.0,
+                sys::EOverlayToStoreFlag::k_EOverlayToStoreFlag_None,
```

Comment:
would be better if we receive the flag as argument, defaulting can lead to unexpected behaviours